semantics.json

```diff
@@ -223,6 +223,39 @@
                   "label": "Seek to after loops amounts",
                   "description": "Enter timecode in the format M:SS"
                 },
+                {
+                  "name": "wrongAnswerParams",
+                  "label": "Wrong Answer Params",
+                  "type": "group",
+                  "importance": "high",
+                  "expanded": true,
+                  "fields": [
+                    {
+                      "name": "isWrongChoice",
+                      "label": "It is wrong choice",
+                      "importance": "low",
+                      "type": "boolean"
+                    },
+                    {
+                      "name": "wrongChoicesAmount",
+                      "type": "number",
+                      "label": "Wrong choices amount number",
+                      "importance": "low",
+                      "description": "When users reach this amount of wrong choices, they will be redirected to a beginning of a task",
+                      "max": 100,
+                      "step": 1,
+                      "default": -1
+                    },
+                    {
+                      "name": "startingTimeOfTask",
+                      "type": "number",
+                      "widget": "timecode",
+                      "importance": "low",
+                      "label": "Starting time of a Task",
```

Comment:
I would not call this task as it might be anything really.
We should work harder to have clearer requirements ;-)

We already have "Seek to after loops amounts" -> best if we stick to this terminology.
"Seek to"

Reply:
Fixed